CONTRIBUTING.md

```diff
@@ -23,14 +21,14 @@ Guidelines for pull requests:
 
 Before creating an icon request, please search to see if someone has requested the icon already. If there is an open request, please add a :+1:.
 
-If the icon has not already been requested, [create an issue](https://github.com/colebemis/feather/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
+If the icon has not already been requested, [create an issue](https://github.com/featherity/featherity/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
 
 ## Bug Reports
 
 Before reporting an issue, please search to see if someone has filed a similar issue before. If there is already an open issue, please add a :+1: and/or leave a comment with additional information.
 
 When creating a new issue make sure to include the following:
-- Version of `Feather` in use. Are you running from source/master? Are you using a released build? Which release?
+- Version of `Featherity` in use. Are you running from source/master? Are you using a released build? Which release?
```

Comment:
This might be confusing to people downloading the icons from the website or using the Figma library.